.config/mkdocs.yml

```diff
@@ -27,11 +27,13 @@ nav:
       - employee-benefits/employee-referral-bonus.md
       - employee-benefits/on-call-stipend.md
   - Company policies:
+      - company-policies/README.md
       - company-policies/expenses.md
       - company-policies/travel-time-tracking-and-expenses.md
       - company-policies/timesheets.md
       - company-policies/security.md
       - Employment policies (US employees):
+          - Employment: company-policies/README.md
```

Comment:
@ContentBrewmaster looks like the company-policies/README.md change is not in this PR. Should this be in another PR?

Reply:
I adjusted the file name.